### PR TITLE
Updated paper with Ken's fixes

### PR DESCRIPTION
**Pull request by aashish24**

### CODE REVIEW

**Comment by aashish24** [comment]
@alvarosan @sankhesh please review

**Comment by aashish24** [comment]
@sankhesh @alvarosan I am thinking that I should merge since these are all good changes and are minor in nature.

File: approach.tex [hunk 1]
```diff
@@ -1,9 +1,9 @@
 \section{Approach}
 \label{approach}
 The new \texttt{vtkGPUVolumeRayCastMapper} uses a ray casting
-technique~\citep{engel_real-time_2006} for volume rendering which is a
-state-of-the-art for volume rendering on modern graphics platforms.
-Algorithmically, at a high level, it is similar to the older version of this
+technique ~\citep{engel_real-time_2006} for volume rendering which
```
Comment (sankhesh) [inline approach.tex:4]: @aashish24 You should not be adding a space before the `~` character. Refer to https://github.com/aashish24/VTKVR/pull/28#discussion_r118722773 for a detailed explanation.

Comment (aashish24) [reply approach.tex:4]: thanks @sankhesh I pushed and merged same PR with the fix.

**Pull request merged**